n/netty/build_info.json

```diff
@@ -2,18 +2,24 @@
     "package_name" : "netty",
     "github_url":"https://github.com/netty/netty.git",
     "package_dir": "n/netty/",
-    "version": "netty-4.1.84.Final",
+    "version": "netty-4.1.92.Final",
     "default_branch": "4.1",
-    "build_script" : "netty-4.1.84.Final_ubi_8.5.sh",
+    "build_script" : "netty_ubi_8.7.sh",
     "docker_build": false,
     "validate_build_script": true,
     "use_non_root_user": false,
 
     "netty-4.1.84.Final" : {
         "build_script" : "netty-4.1.84.Final_ubi_8.5.sh"
     },
-    
+
     "4.1.43.Final-SNAPSHOT" : {
         "build_script" : "netty_build_ppc64le_rhel7.6.sh"
+    },
+    
+     "netty-4.1.92.Final" : {
+        "build_script" : "netty_ubi_8.7.sh"
```

Comment:
@vinodk99 Lets put version string in your latest script as there are multiple scripts in same folder